test/new-e2e/tests/language-detection/language_detection_test.go

```diff
@@ -80,7 +80,10 @@ func (s *languageDetectionSuite) getPidForCommand(command string) string {
 	if err != nil {
 		return ""
 	}
-	return strings.TrimSpace(pid)
+	pid = strings.TrimSpace(pid)
+	// special handling in case multiple commands match
```

Comment:
Does it matter which one is picked? Should it always be the first?

Reply:
I don't think it matters as anything that matches is a `python3` command.